test/unit/tech/flash.test.js

```diff
@@ -30,14 +31,17 @@ test('currentTime', function() {
   // Mock out a Flash instance to avoid creating the swf object
   let mockFlash = {
     el_: {
-      vjs_setProperty: function(prop, val){
+      vjs_setProperty(prop, val){
```

Comment:
These are shorthand for functions. They're different from the methods in classes. Notice how there's a comma between them but not in classes.